tmt/steps/__init__.py

```diff
@@ -103,8 +105,10 @@
 PHASE_ORDER_PREPARE_INSTALL_RECOMMENDS = 75
 
 # Supported steps and actions
-STEPS: list[str] = ['discover', 'provision', 'prepare', 'execute', 'report', 'finish', 'cleanup']
-ACTIONS: list[str] = ['login', 'reboot']
+StepStr = Literal['discover', 'provision', 'prepare', 'execute', 'report', 'finish', 'cleanup']
+STEPS: list[StepStr] = list(typing.get_args(StepStr))
+ActionStr = Literal['login', 'reboot']
```

Comment:
`StepName`, `ActionName`? I really despise when the type becomes a part of the name, `foo_int`, `name_dict`, and so on.